features/gezag-minderjarige/1.3b-is-geadopteerd-met-nederlandse-akte.feature

```diff
@@ -0,0 +1,137 @@
+#language: nl
+
+Functionaliteit: 1.3b - is geadopteerd met nederlandse akte
+  Precondities voor deze vraag:
+  | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                       |
+  | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                       |
+  | 1.3 Is minderjarige naar het buitenland geëmigreerd geweest?           | Nee                                      |
+  | 1.3a Is geboren in het buitenland                                      | Ja                                       |
+
+  Een minderjarige geboren in het buitenland kan zijn geadopteerd met nederlandse akte.
+  
+  Volgens de regels van het Haags Kinderbeschermingsverdrag 1996 volgt uit adopties met een NL akte het Nederlands recht. In dit geval zijn de
+  adoptiefouder(s) de juridische ouder(s).
+  
+  Hoe achterhalen:
+  Bepaal of de persoon op de persoonlijst of in de historiche aktenummers een aktenummer heeft die aangeeft dat de persoon is geadopteerd.
+  
+  Gebruikte velden:
+    - Aktenummer persoon -> 01.81.20
+    - Historische aktenummer -> 51.81.20
+
+    Achtergrond:
+      Gegeven de persoon 'Fabia' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Mohammed' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En de persoon 'Carlo' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is geboren in het buitenland
+      * verblijft in Nederland
+
+  Regel: Kind is geadopteerd met nederlandse akte
+
+    Scenario: Het kind is geadopteerd door één ouder als ouder 1 er is sprake van EenhoofdigOuderlijkGezag
+      Gegeven 'Carlo' is geadopteerd door 'Fabia' als ouder 1
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | EenhoofdigOuderlijkGezag  |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      | ouder.burgerservicenummer        | 000000012                 |
+
+    Scenario: Het kind is geadopteerd door één ouder als ouder 2 er is sprake van EenhoofdigOuderlijkGezag
```

Comment:
Ik zie vast iets over het hoofd maar waarom een scenario met ouder 1 of 2?
Of is dit meer een test op dat je zowel in ouder 1 als in ouder 2 kijkt voor een adoptie-akte?

Reply:
Het is inderdaad meer een test. De uiteindelijk evaluatie wordt gedaan op aktenummers op de persoon. Ouders zijn in de evaluatie irrelevant.

(zelfde geldt ook voor de volgende scenario "... geadopteerd door twee ouders...")

Reply:
Testen voorlopig zo laten staan.